fixtures/common.py

```diff
@@ -38,3 +46,45 @@ def warnings_as_errors():
 def randomness():
     """Ensure a fixed seed for factoryboy"""
     factory.fuzzy.reseed_random("happy little clouds")
+
+
+@pytest.fixture(scope='session', autouse=True)
+def session_indexing_decorator():
+    """
+    Fixture that mocks the reddit object indexer for the test suite by default.
+
+    Decorators require some extra work to patch since they are already applied to functions
+    when a module is loaded. To get around this, we patch the decorator function then
+    reload the relevant modules via importlib.
+    """
+    mock_func = Mock()
```

Comment:
Suggest using `mocker` fixture and call `mocker.Mock()` and `mocker.patch()` instead since those handle `start()`/`stop() shenanigans automatically for you.

Reply:
I believe `mocker` is function-scoped, and therefore not usable in this session-scoped fixture. I'll confirm or deny that

Reply:
Ah I think you're right. That being so, carry on.